docs/superpowers/plans/2026-06-03-withdrawal-history.md

```
@@ -0,0 +1,932 @@
+  withdrawal_request TEXT,
+  synced_at          TEXT NOT NULL DEFAULT (datetime('now')),
+  PRIMARY KEY (stake_address, reward_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_withdrawals_stake_time
+  ON withdrawals (stake_address, delivered_at DESC);
+```
+
+In `wrangler.jsonc`, after the 0002 execute comment line add:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)